nautobot/users/admin.py

```diff
@@ -128,6 +135,30 @@ class TokenAdmin(NautobotModelAdmin):
     list_display = ["key", "user", "created", "expires", "write_enabled", "description"]
 
 
+#
+# SSOBackends
+#
+
+
+class SSOBackendAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = SSOBackend
+        fields = [
+            "name",
+            "backend",
+            "enabled",
+            "configuration",
+        ]
+
+
+@admin.register(SSOBackend)
+class SSOBackendAdmin(NautobotModelAdmin):
+    form = SSOBackendAdminForm
```

Comment:
Is there really a need for a custom form? The fields can limited in the `ModelAdmin`.
```suggestion


@admin.register(SSOBackend)
class SSOBackendAdmin(NautobotModelAdmin):
    fields = ["name", "backend", "enabled", "configuration"]
```